style.css

```diff
@@ -371,12 +384,34 @@ hr {
 
 @media only screen and (max-width: 768px) {
   .menue-btn {
+    position: absolute;
+    margin-top: 0;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: 0;
+    background-color: #6070ff;
+    padding: 13vh 20px;
+    transition: 0.5s;
+    opacity: 0.88;
     display: none;
```

Comment:
- [ ] On some mobile screens, the mobile menu does not cover the entire screen leaving a whitespace at the bottom. Kindly make it fill the entire screen to create a better user experience.
 - [OPTIONAL] Also, it is possible to scroll up and down while the menu is open. It would be nice if the user cannot scroll up and down and leave the menu by accident.
 - Your design:
 - 
<img width="454" alt="image" src="https://user-images.githubusercontent.com/61919617/231157610-8e02a441-701c-4b6f-958b-da58e3aaeff4.png">
<img width="467" alt="image" src="https://user-images.githubusercontent.com/61919617/231157647-51d33f26-624c-4721-81bf-203129b26011.png">